docs/vmc.rst

```diff
@@ -6,6 +6,11 @@ Managing VMC SDDC with Salt
 Things to know about running VMC modules & states
 -------------------------------------------------
 
+
+VMC- Vmware Cloud on AWS which means an instance of the vCloud foundation being executed on AWS bear metal hardware, `You can learn more about VMware Cloud on AWS <https://docs.vmware.com/en/VMware-Cloud-on-AWS/index.html>`_.
```

Comment:
This should be `bare metal` (or `bare-metal`).  :bear:

Reply:
updated